release.md

```diff
@@ -0,0 +1,151 @@
+# Release Guide
+All committer should follow these steps to do release for this repo.
+
+1. Prepare the official release.
+
+- Update the [CHANGES.md](CHANGES.md).
+- Create a new `skywalking-kong-x.y-z.rockspec` file under [rockspec](/rockspec).
+
+2. Package the source release.
+
+```shell
+> export VERSION=x.y.z
+> make release-src
+```
+
+Use SVN to upload the files(tgz, asc and sha512) in the `release` folder to `https://dist.apache.org/repos/dist/dev/skywalking/kong/x.y.z`.
+
+3. Make the internal announcements. Send an announcement mail in dev mail list.
+
+```
+[ANNOUNCE] SkyWalking Kong x.y.z test build available
+
+The test build of x.y.z is available.
+
+We welcome any comments you may have, and will take all feedback into
+account if a quality vote is called for this build.
+
+Release notes:
+
+ * https://github.com/apache/skywalking-kong/blob/vx.y.z/CHANGES.md
+
+Release Candidate:
+
+ * https://dist.apache.org/repos/dist/dev/skywalking/kong/x.y.z/
+ * sha512 checksums
+   - xxxxxxx  skywalking-kong-x.y.z-src.tgz
+
+Release Tag :
+
+ * vx.y.z
+
+Release CommitID :
+
+ * https://github.com/apache/skywalking-kong/tree/xxxxxxxxxx
+
+Keys to verify the Release Candidate :
+
+ * https://dist.apache.org/repos/dist/release/skywalking/KEYS
+
+
+A vote regarding the quality of this test build will be initiated
+within the next couple of days.
+```
+
+4. Wait at least 48 hours for test responses. If there is a critical issue found and confirmed by the PMC, this release should be cancelled.
+
+5. Call for a vote. Call a vote in dev@skywalking.apache.org
+
+```
+[VOTE] Release SkyWalking Kong x.y.z
+
+This is a call for vote to release Apache SkyWalking Kong version x.y.z.
+
+Release notes:
+
+ * https://github.com/apache/skywalking-kong/blob/vx.y.z/CHANGES.md
+
+Release Candidate:
+
+ * https://dist.apache.org/repos/dist/dev/skywalking/kong/x.y.z/
+ * sha512 checksums
+   - xxxxxxx  skywalking-kong-x.y.z-src.tgz
+
+Release Tag :
+
+ * vx.y.z
+
+Release CommitID :
+
+ * https://github.com/apache/skywalking-kong/tree/xxxxxxxxxx
+
+Keys to verify the Release Candidate :
+
+ * https://dist.apache.org/repos/dist/release/skywalking/KEYS
+
+
+Voting will start now (xxxx date) and will remain open for at least 72 hours, Request all PMC members to give their vote.
+[ ] +1 Release this package.
+[ ] +0 No opinion.
+[ ] -1 Do not release this package because....
+
+```
+
+5. Publish release, if vote passed.
```

Comment:
Better to add a step to remove old unrecommended releases in the svn

Reply:
I raise a new PR soon.